### ISSUE TITLE
Why are the thead and tbody rendered as separate tables?

### ISSUE THREAD

**Issue by tacomanator** [AllenFang/react-bootstrap-table#790]
This seems like it might be an issue with accessibility, as the headers are not associated with the content.

**Comment by kakariko-village** [comment]
I totally agree with you. i am working on a project which has a left sidebar. Everytime the left sidebar was toggled, right content will  have margin-right:200 px. However the THead of react boostrap table did not shrink with the right content: http://i.imgur.com/lla8tcW.png

Funny thing is that i set headerStyle={{ width: '100%', background: 'red' }}> and seem like certain part of table header does not work with that

**Comment by AllenFang** [comment]
https://github.com/AllenFang/react-bootstrap-table/issues/164

http://allenfang.github.io/react-bootstrap-table/start.html

**Comment by tacomanator** [comment]
This isn't the same as #164. I'm asking why the structure is like this:
```
<table>
  <thead>
    ...
  </thead>
</table>
<table>
  <tbody>
    ...
  </tbody>
</table>
```

Instead of like this (as it typically would be):

```
<table>
  <thead>
    ...
  </thead>
  <tbody>
    ...
  </tbody>
</table>
```

**Comment by AllenFang** [comment]
https://github.com/AllenFang/react-bootstrap-table/issues/738

Feel free to tell me how to fix it, any solution is very welcome :)
